### PR TITLE
DOC: Add 2019 ITK Community Survey Summary

### PR DESCRIPTION
**Pull request by thewtex**
Co-authored-by: Matt McCormick <matt.mccormick@kitware.com>

### CODE REVIEW

File: src/pages/survey/2019.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+          accelerated processing for large data and machine learning. It is also
+          clear that despite having plenty of processing filters for
+          segmentation and registration, ITK also should continue making ensure
+          adoption of new method to make the state of the art in those domains
+          available for general adoption.
+        </p>
+        <p>
+          When asked about how to strengthen the community, the opinion was
+          expressed to increase the amount of examples and improve the style of
+          the existing examples to adhere to modern style guidelines. Olso
```
Comment (dzenanz) [inline src/pages/survey/2019.js]: Olso -> also

File: src/pages/survey/2019.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+      <br />
+
+      <Typography variant="body1">
+        <h1>Summary</h1>
+        <p>
+          Since 1999, the Insight Toolkit (ITK) has been revolutionizing the way
+          medical images are explored and analyzed. In addition to providing a
+          robust infrastructure for the dissemination of complex algorithms
+          across different fields, ITK has organically created a vibrant and
+          active community in scientific visualization.
```
Comment (dzenanz) [inline src/pages/survey/2019.js]: community in scientific visualization -> community in image processing

**Comment by dzenanz** [comment]
Merge?

**Comment by thewtex** [comment]
Yes :-) :tada: :man_cartwheeling:

**Pull request merged**